.github/workflows/test.yml

```diff
@@ -20,13 +20,13 @@ jobs:
     - name: Set Spark env
       run: |
         export SPARK_LOCAL_IP=127.0.0.1
+        export SPARK_SUBMIT_OPTS="--illegal-access=permit -Dio.netty.tryReflectionSetAccessible=true"
```

Comment:
Added to mitigate issue with identified [here](https://github.com/apache/spark/pull/26552)